README.md

```diff
@@ -83,5 +84,26 @@ EXAMPLE
   $ oxl-cli set-course archive.gz --name "New Course Name"
 ```
 
-_See code: [src/commands/set-course.js](https://github.com/ibm-skills-network/olx-cli/blob/v0.1.5/src/commands/set-course.js)_
+_See code: [src/commands/set-course.js](https://github.com/ibm-skills-network/olx-cli/blob/v0.1.6/src/commands/set-course.js)_
+
+## `olx-cli list-labs COURSEARCHIVEPATH`
+
+List labs attributes
+
+```
+USAGE
+  $ olx-cli list-labs COURSEARCHIVEPATH
+
+ARGUMENTS
+  COURSEARCHIVEPATH  Path to OXL course archive
+
+DESCRIPTION
+  Lists all labs found in a course along with their tool types. Labs are found either thorugh lti_consumer xml blocks or iframe tags in html
+
+EXAMPLE
+  $ oxl-cli list-labs archive.gz
```

Comment:
docs: may want to include some sample output and mention in the description that the output will be in JSON format